src/sdl_utils.cpp

```diff
@@ -82,13 +82,8 @@ SDL_Surface_Ptr create_surface_32( int w, int h )
 void render_fill_rect( const SDL_Renderer_Ptr &renderer, const SDL_Rect &rect,
                        Uint32 r, Uint32 g, Uint32 b )
 {
-    if( alt_rect_tex_enabled ) {
-        SetTextureColorMod( alt_rect_tex, r, g, b );
-        RenderCopy( renderer, alt_rect_tex, nullptr, &rect );
-    } else {
-        SetRenderDrawColor( renderer, r, g, b, 255 );
-        RenderFillRect( renderer, &rect );
-    }
+    SetRenderDrawColor( renderer, r, g, b, 255 );
+    RenderFillRect( renderer, &rect );
```

Comment:
Why isn't this using the new renderer classes?

Reply:
I wasn't sure how to do it right. It was using a global method which in turn used global variables. 

Now that I'm lookin at references of this method, it is possible to pass geometry renderer into `cata_tiles`, and then into `pixel_minimap`, which would eliminate any need in this method at all.